src/webapp/components/survey-list/table/SurveyListTable.tsx

```diff
@@ -224,6 +236,31 @@ export const SurveyListTable: React.FC<SurveyListTableProps> = ({
                                         </TableCell>
                                     )}
 
+                                    {[
```

Comment:
create and export array in domain like SURVEYS_WITH_CHILD_COUNT or with appropriate name and use it everywhere.